src/engraving/dom/measure.cpp

```diff
@@ -1970,6 +1970,21 @@ bool Measure::isFinalMeasureOfSection() const
     return false;
 }
 
+LayoutBreak* Measure::isFinalMeasureOfSection()
```

Comment:
Could this be renamed to something like `nextSectionBreak`?

Reply:
> Could this be renamed to something like `nextSectionBreak`?

I was thinking, what could be good name. 
Because `nextSectionBreak` may be confusing too. One could expect next section break from any measure (not only from last in section).

May be something like `sectionBreakElement(bool checkNextFrames)`?

Reply:
I renamed it as described above. 
@miiizen, please, check, if it could be.